Gemfile.lock

```diff
@@ -2,10 +2,11 @@ GEM
   remote: https://rubygems.org/
```

Comment:
Having to explicitly change the Gemfile.lock file is the point.  It allows releasing a version which has been tested with known dependencies.  If there isn't a lock file then someone will get the latest releases of the dependencies not the versions that the release was tested with.  It's fine to update the dependencies, the lock file is making sure that people know they are updating them.

It's also a Ruby convention to include the lock file.